api/src/main/java/com/datatorrent/api/LocalMode.java

```diff
@@ -18,34 +18,62 @@
  */
 package com.datatorrent.api;
 
-import java.util.Iterator;
-import java.util.ServiceLoader;
-
+import org.apache.apex.api.Launcher;
 import org.apache.hadoop.conf.Configuration;
 
 /**
  * Local mode execution for single application
  *
+ * @deprecated
```

Comment:
I see. I will try to flip it and make LocalMode extend the EmbeddedAppLauncher. Still including it in the same inheritance hierarchy so that the implementation does not have to be repeated.